op-e2e/inbox/inbox_test.go

```diff
@@ -0,0 +1,168 @@
+	Bin: "60a0604052348015600e575f5ffd5b506040516104fc3803806104fc8339818101604052810190602e9190606d565b8060808181525050506093565b5f5ffd5b5f819050919050565b604f81603f565b81146058575f5ffd5b50565b5f815190506067816048565b92915050565b5f60208284031215607f57607e603b565b5b5f608a84828501605b565b91505092915050565b6080516104526100aa5f395f60aa01526104525ff3fe608060405260043610610033575f3560e01c80631ccbc6da146100375780634581a92014610061578063638ba9e91461007d575b5f5ffd5b348015610042575f5ffd5b5061004b6100a7565b60405161005891906101c6565b60405180910390f35b61007b60048036038101906100769190610240565b6100ce565b005b348015610088575f5ffd5b506100916101a9565b60405161009e91906101c6565b60405180910390f35b5f7f0000000000000000000000000000000000000000000000000000000000000000905090565b5f824990505f5f1b8103610117576040517f08c379a000000000000000000000000000000000000000000000000000000000815260040161010e90610310565b60405180910390fd5b5f5f1b840361015b576040517f08c379a00000000000000000000000000000000000000000000000000000000081526004016101529061039e565b60405180910390fd5b5f5f54905060015f5461016e91906103e9565b5f819055508183827f8b7a21215282409938287ae262331bfe6411d35d3d46aa7e505ef02000524ac260405160405180910390a45050505050565b5f5481565b5f819050919050565b6101c0816101ae565b82525050565b5f6020820190506101d95f8301846101b7565b92915050565b5f5ffd5b5f819050919050565b6101f5816101e3565b81146101ff575f5ffd5b50565b5f81359050610210816101ec565b92915050565b61021f816101ae565b8114610229575f5ffd5b50565b5f8135905061023a81610216565b92915050565b5f5f5f60608486031215610257576102566101df565b5b5f61026486828701610202565b93505060206102758682870161022c565b92505060406102868682870161022c565b9150509250925092565b5f82825260208201905092915050565b7f45746853746f72616765436f6e74726163743a206661696c656420746f2067655f8201527f7420626c6f622068617368000000000000000000000000000000000000000000602082015250565b5f6102fa602b83610290565b9150610305826102a0565b604082019050919050565b5f6020820190508181035f830152610327816102ee565b9050919050565b7f45746853746f72616765436f6e74726163743a206661696c656420746f2067655f8201527f7420626c6f62206b657900000000000000000000000000000000000000000000602082015250565b5f610388602a83610290565b91506103938261032e565b604082019050919050565b5f6020820190508181035f8301526103b58161037c565b9050919050565b7f4e487b71000000000000000000000000000000000000000000000000000000005f52601160045260245ffd5b5f6103f3826101ae565b91506103fe836101ae565b9250828201905080821115610416576104156103bc565b5b9291505056fea2646970667358221220843189a0c55b11adcf6318b93e4bb7cddf32d681f005f2ceb5eec50aa5cf3c4d64736f6c634300081c0033",
+}
+
+// BatchInboxMetaData contains all meta data concerning the L1Block contract.
+var BatchInboxMetaData = &bind.MetaData{
```

Comment:
I wonder if there is any better way than hardcode the abi and bin?

Reply:
add comments on how to generate those ABI and Bin.